src/exporter/GpifWriter.ts

```diff
@@ -1335,9 +1335,11 @@ export class GpifWriter {
                                 }
 
                                 const line = lines[l];
+                                const beatLyricsLine = beat.lyrics[l] || '';
```

Comment:
Null Coalescing would be better in such scenarios
```suggestion
                                const beatLyricsLine = beat.lyrics[l] ?? '';
```